openmicroscopy/openmicroscopy
# prevent from editing map annotations in batch_annotation pane

# What this PR does

partial rebase from https://github.com/openmicroscopy/openmicroscopy/pull/4923
only commits related to metadata panel

# Testing this PR

1. Login to web
2. Select multiple images
3. check if editing mode is not available on key-value pairs. Maps should be present
